confidence/build.gradle.kts

```diff
@@ -0,0 +1,119 @@
+plugins {
+    id("com.android.library")
+    id("org.jetbrains.kotlin.android")
+    id("maven-publish")
+    id("signing")
+    kotlin("plugin.serialization").version("1.8.10").apply(true)
+}
+
+val providerVersion = project.extra["version"].toString()
+
+object Versions {
```

Comment:
Ideally we want to keep these versions set in the project gradle file to share them

Reply:
exactly.

Reply:
yes.